Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py

```diff
@@ -105,12 +106,11 @@ def get_access_token_data(self):
     def get_authentication_string(self):
         return f'Bearer {self.access_token}'
 
-    def get_blob_data_request(self, blob_url, timeout=10):
+    def get_blob_data_request(self, blob_url, timeout=TIMEOUT):
```

Comment:
This will not work. `timeout` will always be set to `15`.
You'll need to set it to `None`, and access the global `TIMEOUT` in case `timeout` is set to `None`.